dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/CDIFlowsTests.java

```diff
@@ -84,6 +89,7 @@ public static void testCleanup() throws Exception {
      * Ensure that each app is running with the appropriate provider so that
      * we don't accidentally run both apps with a single provider.
      */
+    @SkipForRepeat(SkipForRepeat.EE11_FEATURES)
```

Comment:
There is a Mojarra 4.1 version: https://mvnrepository.com/artifact/org.glassfish/jakarta.faces/4.1.0 I'd like us to use this for testing unless there is another reason we're skipping these tests.